windows/container.go

```diff
@@ -174,6 +174,11 @@ func (c *container) Pid() uint32 {
 	return c.ctr.Pid()
 }
 
+func (c *container) Ps(ctx context.Context) ([]uint32, error) {
+	// TODO (ehazlett): support finding all processes in windows container
```

Comment:
Signal only support signaling processes created via the runtime yes.

I just meant that there can be some confusion in the value returned in the pid array as it won't match what the runtime returned for processes that were created by it.